packages/esm-ward-app/src/config-schema.ts

```diff
@@ -132,66 +168,115 @@ export const configSchema: ConfigSchema = {
+            _description:
+              'The color of the summary tag. See https://react.carbondesignsystem.com/?path=/docs/components-tag--overview for a list of supported colors',
+            _default: null,
+          },
+          tags: {
+            _type: Type.Array,
+            _description: `An array specifying concept sets and color. Observations with coded values that are members of the specified concept sets will be displayed as their own tags with the specified color. Any observation with coded values not belonging to any concept sets specified will be summarized as a count in the summary tag. If a concept set is listed multiple times, the first matching applied-to rule takes precedence.`,
+            _default: [],
+            _elements: {
+              color: {
+                _type: Type.String,
+                _description:
+                  'Color of the tag. See https://react.carbondesignsystem.com/?path=/docs/components-tag--overview for a list of supported colors.',
+              },
+              appliedToConceptSets: {
+                _type: Type.Array,
+                _description: `The concept sets which the color applies to. Observations with coded values that are members of the specified concept sets will be displayed as their own tag with the specified color. If an observation's coded value belongs to multiple concept sets, the first matching applied-to rule takes precedence.`,
+                _elements: {
+                  _type: Type.UUID,
+                },
+              },
             },
           },
         },
       },
     },
   },
+  wards: {
+    _description: 'Configuration of what type of ward to use at different ward locations.',
+    _type: Type.Array,
+    _default: [{ id: 'default-ward' }],
+    _elements: {
+      id: {
+        _type: Type.String,
+        _description:
+          'The ward type to use. Currently, "default-ward" and "maternal-ward" are supported. This string also serves as the extension slot name for the ward view.',
+      },
+      appliedTo: {
+        _type: Type.Array,
+        _description:
+          'Optional. Conditions under which this card definition should be used. If not provided, the configuration is applied to all wards.',
+        _elements: {
+          location: {
+            _type: Type.UUID,
+            _description: 'The UUID of the location. If not provided, applies to all wards.',
+            _default: null,
+          },
+        },
+      },
+    },
+  },
 };
 
 export interface WardConfigObject {
-  wardPatientCards: WardPatientCardsConfig;
-}
-
-export interface WardPatientCardsConfig {
-  obsElementDefinitions: Array<ObsElementDefinition>;
-  pendingItemsDefinitions: Array<PendingItemsDefinition>;
-  identifierElementDefinitions: Array<IdentifierElementDefinition>;
-  addressElementDefinitions: Array<AddressElementDefinition>;
-  cardDefinitions: Array<WardPatientCardDefinition>;
+  patientCardElements: {
```

Comment:
Potential rabbit-hole... I guess I was thinking that each WardView would have it's own config object?  When we get to the point where a custom WardView can be defined outside the Ward App ESM, is there a way for the developers of new Ward Views to define new config parameters without adding to the Ward View config?  (For discussion, let's not let this be a blocker for merging)

Reply:
For a custom WardView outside this ESM, it can have patient cards with a mixture of elements defined in this ESM and outside it. I think ideally we'd have the elements configurations be tied to whichever ESM it's defined in. We'll want to be careful as what `useConfig()` returns is dependent on which ESM / extension the component is in. I haven't done something like this before, but it feels doable.

Reply:
Probably the _best_ solution here is for the ward app to consume whatever config options it needs to consume and pass those as props to the elements or something along those lines...